src/map/entities/charentity.cpp

```diff
@@ -2079,6 +2079,14 @@ void CCharEntity::OnItemFinish(CItemState& state, action_t& action)
     auto* PTarget = static_cast<CBattleEntity*>(state.GetTarget());
     auto* PItem   = state.GetItem();
 
+    if (!PItem->isType(ITEM_EQUIPMENT) && (PItem->getQuantity() < 1 || PItem->getReserve() > 0))
+    {
+        ShowWarning("OnItemFinish: %s attempted to use reserved/insufficient %s (%u).", this->getName(), PItem->getName(), PItem->getID());
```

Comment:
I believe `this->getName()` should be `this->GetName()`

getName() looks like an item string

Reply:
We renamed entity and zone GetName's to getName, and we plan on doing the same for everything else so all member functions (and their bindings) start with lower case letters 👍

Reply:
ahh ok, thank you!